homeassistant/components/yeelightsunflower/light.py

```diff
@@ -68,9 +73,9 @@ def brightness(self):
         return int(self._brightness / 100 * 255)
 
     @property
-    def hs_color(self):
-        """Return the color property."""
-        return self._hs_color
+    def rgb_color(self):
```

Comment:
This isn't correct. A light entity that supports color needs to implement the `hs_color` property.

Reply:
Ah, I didn't know that. 
Can you help me understand please. When I look at https://developers.home-assistant.io/docs/core/entity/light I don't see the requirement. (I don't see rgb_color either, but I'm not implementing many of those properties).

It works without it, but I'll make it convert the RGB to HS and implement that method.